### ISSUE TITLE
Required filename with --stdin and --stdout flag

### ISSUE THREAD

**Issue by danilobuerger** [tjko/jpegoptim#110]
Starting with 1.4.7 when using `jpegoptim --stdout --stdin` I get the following error message:
```
jpegoptim: file argument(s) missing
Try 'jpegoptim --help' for more information.
```
I do not want to specify a filename as I am using stdin and stdout. This worked in 1.4.6

**Comment by tjko** [comment]
Thanks for pointing this out.  This should now be fixed (commit: 77fcf640624402377e785f92f6b5f3f375befe1f) ?

**Comment by danilobuerger** [comment]
Thanks!
